artifactory/commands/golang/archive.go

```diff
@@ -98,10 +98,14 @@ func archiveProject(writer io.Writer, dir, mod, version string) error {
 	//ignore, gitIgnoreErr := gitignore.NewFromFile(sourcePath + "/.gitignore") ??
 	var files []File
 
-	err := filepath.Walk(dir, func(filePath string, info os.FileInfo, err error) error {
-		relPath, err := filepath.Rel(dir, filePath)
+	err := filepath.Walk(dir, func(filePath string, info os.FileInfo, err error) (currentErr error) {
```

Comment:
The function does not use defers. I think there is no need to use `currentErr` here.